meilisearch-release-changelog/v1.8.0.md

```diff
@@ -0,0 +1,219 @@
+# v1.8.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.8 introduces new changes and optimizations related to the Hybrid search with the addition of new models and embedders like REST embedders and the Ollama model. This version also focuses on stability by adding more security around the search requests. Finally, we introduce the negative operator to exclude specific terms from a search query.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
```

Comment:
Yes still true. I mean the situation did not change, we do the same.
We update the all integrations to ensure there is no breaking -> you can still use the SDK with the new version of Meilisearch. But we don't implement ALL features in ALL SDKs. And even before the layoff we did not implement all the features in all SDKs

Reply:
ok, sounds good! Just wanted to check since I don't always review these default bits and thought it was a good idea to verify everything is still accurate.